app/grandchallenge/algorithms/models.py

```diff
@@ -39,6 +42,8 @@
 DEFAULT_INPUT_INTERFACE_SLUG = "generic-medical-image"
 DEFAULT_OUTPUT_INTERFACE_SLUG = "generic-overlay"
 
+JINJA_ENGINE = sandbox.SandboxedEnvironment()
```

Comment:
Shouldn't we use `ImmutableSandboxedEnvironment` to prevent modification of the objects? I think so as we send `output.value` directly and we do not want to allow accidental modification of that.